h2o-extensions/target-encoder/src/main/java/ai/h2o/targetencoding/TargetEncoder.java

```diff
@@ -544,7 +544,7 @@ public Frame applyTargetEncoding(Frame data,
      * @param data dataset that will be used as a base for creation of encodings .
      * @param targetColumnName name of the column with respect to which we were computing encodings.
      * @param columnToEncodingMap map of the prepared encodings with the keys being the names of the columns.
-     * @param dataLeakageHandlingStrategy see TargetEncoding.DataLeakageHandlingStrategy //TODO use common interface for stronger type safety.
+     * @param dataLeakageHandlingStrategy see TargetEncoder.DataLeakageHandlingStrategy //TODO use common interface for stronger type safety.
```

Comment:
@deil87 Is this a name change or just fixing a typo?

Reply:
@ledell this is just fixing a typo